frontend/src/components/MetricsSummary.tsx

```diff
@@ -21,7 +21,7 @@ export const MetricsSummary: FC<MetricsProps> = ({
       <>
         <div className={classes(css.dash)} />
         <div className={classes(css.dataNumberWrapper)}>
-          {value}
+          {Math.round(Number(value))}
```

Comment:
Just to note: we'll need a null check once this is rebased, since in master the value can be null as well
```suggestion
          {value !== null && Math.round(Number(value))}
```

Reply:
Where was this problem, so that rounding was needed? I forgot and couldn't find it anymore

Not needing this would be great

Reply:
I remember some metrics would get too large with the new scaling if decimals were shown too. Maybe it was in task overview. 
One thought is, instead of modifying this component, maybe we could do rounding in places where it's needed.

Reply:
Oh right the content can overlap in task map. What do you think about only rounding there and removing the statement from MetricsSummary?